### PR TITLE
Modified README.md and examples/README.md

### PR DESCRIPTION
**Pull request by Hridayshaha**
I updated **README.md and examples/README.md files**. README.md file I change "It is an open-source repository and feel free to add documentation, examples, and other references when necessary." to  It is an open-source repository and feels free to add documentation, examples, and other references when necessary. And also update some text and commas in this file examples/README.md. 

This is my first open-source contribution as a new developer. Thanks, stack learner to create a way where we can contribute to opensource. It increased my confidence now by 10000% to contribute to others people's Open Souce project.

### CODE REVIEW

**Review by mamutalib — COMMENTED** [review]
Great Work

**Comment by Hridayshaha** [comment]
> Great Work

Thanks a lot, it will be inspire me a lot,

File: examples/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+Feel free to create examples & components. You can showcase your work directly here. For example, you have designed a card component or a cute nice alert box. You can share your code here with other learners. They can take motivation from your code and design. To get your example accepted, you have to follow the instructions below.
 
 **Instructions:**
 
--   Create a directory for your example project. Give it a meaningful name, use lowercase letters when naming your directory.
--   Your directory must contain a `README.md` file describing your project, requirements, necessary screenshots and other resources.
--   Must follow a good directory structure like.
+- Create a directory for your example project. When naming your directory, use lowercase letters and give it a meaningful name.
+- Your directory must contain a `README.md` file describing your project, requirements, necessary screenshots, and other resources.
+- Must follow a good directory structure.
```
Comment (FahimFBA) [inline examples/README.md:9]:
```diff
- Must follow a good directory structure.
+ You should follow the directory structure.

**Review by FahimFBA — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Stack Learner - Web Design Master Class
 
-In this repository, you will find source codes, explanations, and other resources for **Web Design Master Class**. It is an open-source repository and feel free to add documentation, examples, and other references when necessary. Please create issues when you get confused or if something does not work. Other members will help you to solve your issues.
+In this repository, you will find source codes, explanations, and other resources for **Web Design Master Class**. It is an open-source repository and feels free to add documentation, examples, and other references when necessary. Please create issues when you get confused or if something does not work. Other members will help you to solve your issues.
```
Comment (mamutalib) [inline README.md:3]: #

Comment (FahimFBA) [reply README.md:3]: Your requested change is unnecessary considering my requested changes. A repository can't feel free to add anything; A human does.

**Review by mrhm-dev — APPROVED** [review]

**Review by mrhm-dev — APPROVED** [review]

**Pull request merged**